presentation/src/main/java/com/knocklock/presentation/lockscreen/LockScreenNotificationListener.kt

```diff
@@ -39,7 +37,7 @@ class LockScreenNotificationListener : NotificationListenerService() {
             val largeIcon = notification.getLargeIcon()
 
             scope.launch {
-                notificationRepository.insertNotification(
+                insertNotificationUseCase.invoke(
```

Comment:
```suggestion
                insertNotificationUseCase(...)
```
operator 함수가 있다면 invoke()없이 바로 사용하면 어떨까요?

Reply:
viewModel에서는 그렇게 했는데 빼먹었네영 반영할께용
~

Reply:
@KimHance [a7c2202](https://github.com/Knock-Lock/Knock-Lock/pull/50/commits/a7c220228d7413f15ca7278d71f8ae2fc478fbb4) 반영했습니당